src/ray/protobuf/core_worker.proto

```diff
@@ -71,6 +71,15 @@ message ReturnObject {
   int64 size = 6;
 }
 
+message StealWorkRequest {
+  int64 max_tasks_to_steal = 1;
+}
+
+message StealWorkReply {
+  int64 number_of_tasks_stolen = 1;
+  repeated TaskSpec tasks_stolen = 2;
+}
+
 message PushTaskRequest {
```

Comment:
To take full advantage of vectorization, don't we need to push a list of tasks to the worker instead of one per RPC?

Reply:
We're still getting some improvement from grpc batching, but yeah, long-term we should consider sending multiple tasks to a worker in a single RPC. Right now it's a little tricky to do that without adding latency for other tasks because the return values of the task are also sent back in the reply.

Reply:
Makes sense. You could do something like best effort vectorization of replies (send replies back along a separate reply path). This is probably something that would make sense once we have gRPC streaming support.